TestAdapterforBoost.TestDev17.yml

```diff
@@ -34,6 +34,10 @@ resources:
     type: git
     name: VCLS-Extensions
     ref: refs/heads/dev17
+parameters:
```

Comment:
So sign type is always real now? Or are we still using the TAfGTSignType variable in the ADO pipeline variables?

Reply:
I'm going to delete the TAfBTSignType variable (once this goes through since it's no longer used). I changed from a pipeline variable to a parameter (so you can still either pass in a different value when you call this, or can change it manually in the UI when you try to queue a build). Real signing is just the default.

Reply:
please set to test by default instead for quicker test runs by default. Documentation for these pipelines state to switch all variables to True/Real for official builds+insertions.